configs/components/openssl.rb

```diff
@@ -127,19 +128,22 @@
       #{target} \
       #{sslflags} \
       no-camellia \
-      enable-seed \
       enable-tlsext \
       enable-rfc3779 \
-      enable-cms \
       no-md2 \
       no-mdc2 \
-      no-rc5 \
+      no-weak-ssl-ciphers \
+      no-dtls \
+      no-dtls1 \
+      no-rc4 \
+      -DOPENSSL_NO_RC4 \
+      no-idea \
+      no-seed \
       no-ec2m \
-      no-gost \
-      no-srp \
```

Comment:
These too seem to be getting disabled by no-weak-ssl-ciphers. In my testing these wont show up and our "SSL Implementation Reference" document recommends disabling these. Unless you are saying there are some developments/research suggesting these can be used.